tests/test_exceptions.py

```diff
@@ -72,3 +74,51 @@ def __call__(self):
     with_retry(FailerUntil(2).__call__)()
     with pytest.raises(SomeException):
         with_retry(FailerUntil(3).__call__)()
+
+
+def test_trino_query_error_available():
```

Comment:
Are you aware that test_client.py has similar test (`test_trino_fetch_error`)?

Reply:
It kind of is but it doesn’t check all fields and serves as a semi-integration one. Should I reuse JSON from there or leave this change as is?

Reply:
Could you rename this test method? Maybe, `test_trino_query_syntax_error` or something.